src/test/java/uk/gov/digital/ho/pttg/application/HmrcHateoasClientTest.java

```diff
@@ -556,4 +522,41 @@ public void getMatchResource_hmrcOverRateLimitException_notCaught() {
         assertThatThrownBy(() -> client.getMatchResource(individual, "some access token"))
                 .isEqualTo(hmrcOverRateLimitException);
     }
+
+    @Test
+    public void getMatchResource_noMatch_logUnsuccessful() {
+        given(mockHmrcCallWrapper.exchange(any(), eq(POST), any(HttpEntity.class), any(ParameterizedTypeReference.class)))
+                .willThrow(new ApplicationExceptions.HmrcNotFoundException(""));
+
+        try {
+            client.getMatchResource(individual, "any access token");
+        } catch (ApplicationExceptions.HmrcNotFoundException e) {
+            // swallowed as not of interest to test
+        }
+
+        then(mockAppender)
+                .should(atLeastOnce())
+                .doAppend(logArgumentCaptor.capture());
+
+        assertThat(findLog("Unsuccessfully matched individual NR123456C").getArgumentArray())
+                .contains(new ObjectAppendingMarker(EVENT, HMRC_MATCHING_UNSUCCESSFUL));
+    }
+
+    private LoggingEvent findLog(String message) {
```

Comment:
It might be better to search on the event - then the individual test can verify the message if required (e.g. if it contains some relevant data).